app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/feature/AppTpRemoteFeatures.kt

```diff
@@ -134,3 +179,80 @@ class AppTpRemoteFeaturesStore @Inject constructor(
         private const val PREFS_FILENAME = "com.duckduckgo.vpn.atp.remote.features.v1"
     }
 }
+
+@ContributesMultibinding(AppScope::class)
+class AppTpTDSPixelsPlugin @Inject constructor(private val inventory: FeatureTogglesInventory) : MetricsPixelPlugin {
+
+    override suspend fun getMetrics(): List<MetricsPixel> {
+        val activeToggle = inventory.activeAppTpTdsFlag() ?: return emptyList()
```

Comment:
As this is doing disk IO I'd wrap it in `dispatchers.io()`. Probably you can wrap the entire function
```kotlin
override suspend fun getMetrics(): List<MetricsPixel> = withContext(dispatchers.io() {
  val activeToggle = ...
  return@withContext listOf(...)
}

```